src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -1272,6 +1282,7 @@ lVbRuntimePlus:
             Return New VisualBasicCommandLineArguments With
             {
                 .IsScriptRunner = IsScriptRunner,
+                .InteractiveMode = InteractiveMode Or (IsScriptRunner AndAlso sourceFiles.Count = 0),
```

Comment:
This is what the CSharp command-line parser does (without comment, though some explanation might be helpful). If the check for source files is not present, VBI fails to start.

"Or" might better be "OrElse", and I think the casing of "InteractiveMode" should be "interactiveMode".